src/robusta/core/playbooks/common.py

```diff
@@ -17,7 +17,7 @@ def get_resource_events_table(table_name: str, kind: str, name: str, namespace:
         headers = ["reason", "type", "time", "message"]
         rows = [
             [event.reason, event.type, parse_kubernetes_datetime_to_ms(event.lastTimestamp), event.message]
-            for event in event_list.items
+            for event in event_list.items if event.lastTimestamp
```

Comment:
I had an event that had no last timestamp and it threw here, it was weird

Reply:
yeah, we saw that.
Good catch

Reply:
Do we really want to remove events without timestamp?
Would it be better to add them with some fixed timestamp?
WDYT?

Reply:
I'll second that thought. I recently got a report from a user about this happening on their environment. 

I think the best solution is to add them to the table and either leave the time column empty or write `time missing`

Reply:
the only options are making a magic timestamp that we can understand from the UI is missing a timestamp, or making a column in the issues table for no timestamp, also sounds like a waste for an uncommon event

the checking all three timestamps is easily do-able though

Reply:
@arikalon1 , Im not sure the previous event would work in this case since there is no guarantee its in chronological order,
also having a bunch of events in the UI with the time of 0 could look buggy